index.js

```diff
@@ -0,0 +1,76 @@
+document.addEventListener("DOMContentLoaded",()=>{
+
+fetch("https://ghibliapi.herokuapp.com/films")
+.then(response => response.json())
+.then(response => {
+  getFilms(response)
+})
+
+let select = document.querySelector("select")
+select.addEventListener("change",(e)=>{
+  let words = document.querySelector("#words")
+  words.value = ""
+  let info = document.querySelector("#info")
+  let index = arrjson[e.target.selectedIndex]
+  fetch(`https://ghibliapi.herokuapp.com/films/${index}`)
```

Comment:
I'm glad you are making a second request to `/films/<film-id>` to get the movie details. It would be nicer if you break it down all the functionality into a few different functions though. Also you should always have `.catch()` with `fetch`.